### ISSUE TITLE
`render()` statement without parameter caused NPE in controller class enhancement logic

### ISSUE THREAD

**Issue by greenlaw110** [actframework/actframework#166]
The code:

```java
    @GetAction("/x")
    public void x() {
        render();
    }
```

The error stack:

```
java.lang.NullPointerException: null
	at act.controller.bytecode.HandlerEnhancer$Transformer.mergeRenderLineBreaks(HandlerEnhancer.java:163)
	at act.controller.bytecode.HandlerEnhancer$Transformer.mergeRenderLineBreaks(HandlerEnhancer.java:154)
	at act.controller.bytecode.HandlerEnhancer$Transformer.<init>(HandlerEnhancer.java:133)
	at act.controller.bytecode.HandlerEnhancer.transform(HandlerEnhancer.java:121)
	at act.controller.bytecode.HandlerEnhancer.visitEnd(HandlerEnhancer.java:105)
	at act.asm.MethodVisitor.visitEnd(MethodVisitor.java:877)
	at act.asm.ClassReader.readMethod(ClassReader.java:1021)
	at act.asm.ClassReader.accept(ClassReader.java:693)
	at act.asm.ClassReader.accept(ClassReader.java:506)
	at act.app.AppClassLoader.asmEnhance(AppClassLoader.java:477)
	at act.app.AppClassLoader.enhance(AppClassLoader.java:464)
	at act.app.AppClassLoader.loadAppClass(AppClassLoader.java:441)
	at act.app.AppClassLoader.loadClass(AppClassLoader.java:230)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.osgl.Osgl.classForName(Osgl.java:5216)
	at act.handler.builtin.controller.impl.ReflectedHandlerInvoker.<init>(ReflectedHandlerInvoker.java:105)
	at act.handler.builtin.controller.impl.ReflectedHandlerInvoker.createControllerAction(ReflectedHandlerInvoker.java:490)
	at act.Act$Mode.createRequestHandler(Act.java:130)
	at act.handler.builtin.controller.RequestHandlerProxy.generateHandlers(RequestHandlerProxy.java:346)
	at act.handler.builtin.controller.RequestHandlerProxy.ensureAgentsReady(RequestHandlerProxy.java:287)
	at act.handler.builtin.controller.RequestHandlerProxy.sessionFree(RequestHandlerProxy.java:227)
	at act.handler.DelegateRequestHandler.sessionFree(DelegateRequestHandler.java:76)
	at act.app.ActionContext.resolve(ActionContext.java:674)
	at act.route.Router$ContextualHandler.handle(Router.java:1244)
	at act.xio.NetworkHandler$1.run(NetworkHandler.java:122)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by greenlaw110** [comment]
The workaround is to remove the line `render()`:

```java
    @GetAction("/x")
    public void x() {
    }
```

**Comment by greenlaw110** [comment]
Fixed in #170ab09
